sdk/data/aztables/autorest.md

```diff
@@ -11,7 +11,7 @@ clear-output-folder: false
 output-folder: internal
 tag: package-2019-02
 credential-scope: none
-use: "@autorest/go@4.0.0-preview.36"
+use: "C:/github.com/autorest.go"
 module-version: 0.5.1
```

Comment:
Does this need to be updated now, or will that happen once the official code generator has been released along with `azcore`?

Reply:
Once the autorest release is tagged I will update this